astro/src/content/blog/understanding-the-authorization-code-grant.mdx

```diff
@@ -0,0 +1,127 @@
+
+
+![Private Division's custom login page with video game highlights on the left and a login form on the right.](/img/blogs/video-auth-code-grant/1-video-auth-code-grant.png)
+
+#### Why the Redirect is Crucial
+
+The redirect serves two main purposes: security and features.
+
+1. **Security**:
+    
+    
+    - By redirecting users to FusionAuth's hosted login pages, we make sure that applications delegating authentication to FusionAuth don't need to handle sensitive information like credentials.
+    - This minimizes the risk of sensitive data being spread across multiple applications.
+    
+    <Aside type="note">
+    The application simply sends the user to FusionAuth. FusionAuth performs the checks and then returns the necessary information to the application.
+    </Aside>
+2. **Features**:
+    
+    
+    - Delegating authentication to FusionAuth enables us to offer new features without burdening your application.
+    - Features like [profile](/docs/lifecycle/manage-users/account-management/updating-user-data) management, [Multi-Factor Authentication (MFA)](/docs/lifecycle/authenticate-users/multi-factor-authentication), [session management](/docs/lifecycle/authenticate-users/logout-session-management), etc., can be added through FusionAuth or its APIs with minimal complexity.
+    
+    <Aside type="note">
+    New features can be added via FusionAuth or its APIs, keeping the application blissfully unaware of the complexities involved.
+    </Aside>
+
+## Implementing the Authorization Code Grant with FusionAuth
+
+To implement this, we'll follow easily configured steps, making sure the application is well-integrated with FusionAuth.
+
+### Step 1: Setting Up the Authorization Request
+
+Start by building the [authorization code grant request](https://fusionauth.io/docs/lifecycle/authenticate-users/oauth/endpoints#authorization-code-grant-request). This URL will redirect the user to FusionAuth's login page.
+
+```http
+GET /oauth2/authorize?client_id={client_id}&redirect_uri={redirect_uri}&response_type=code&tenantId={tenantId}&scope={scope}
+```
+
+**Replace the placeholders**:
+
+- `client_id`: Your Application's Client Id.
+- `redirect_uri`: The URI where the user will be redirected after logging in.
+- `scope`: The permissions your application needs, like `openid` and `offline_access`.
+- `response_type`: Always `code` in this case.
+- `tenantId`: The unique Tenant Id used for applying the proper theme.
+
+<Aside type="tip">
+Make sure to URL encode the query parameters.
+</Aside>
+
+### Step 2: Handling the Redirect
+
+When the user logs in successfully, FusionAuth will redirect them back to your `redirect_uri` with an [authorization code and state](/docs/lifecycle/authenticate-users/oauth/endpoints#response). Your application needs to extract these parameters.
```

Comment:
Just a suggestion: 
```suggestion
When the user logs in successfully, FusionAuth will redirect them back to your `redirect_uri` with an [authorization code and state](/docs/lifecycle/authenticate-users/oauth/endpoints#response). Your application needs to handle the callback and extract these parameters.
```
This example seems a little weird to me. The application needs to be able to handle this, but it almost seems like your application needs to make this call which is not the case.